### ISSUE TITLE
Where is verifyUser invoked?

### ISSUE THREAD

**Issue by corderophi678** [FrontendMasters/api-design-node-v2#4]
In `/src/api/modules/auth.js` the `signin` function has a comment that says "req.user will be there from the middleware verify user." But it's unclear to me where verifyUser is invoked in order to put the user's id on the req object. Any help would be super appreciated.

**Comment by timiscoding** [comment]
I use it like so `app.use('/signin', verifyUser(), signin)` in place of this line https://github.com/FrontendMasters/api-design-node-v2/blob/733c5e1d8144f244bc6da539706eb44fa8896da9/src/server.js#L14

Auth is actually quite broken but luckily the instructor doesn't use it, he just bypasses it by creating a new user or disabling auth altogether. 

[Check out my branch](https://github.com/timiscoding/api-design-node-v2/commit/06fcda3fe4b34f27f2297f1fc017b1116d8fa285) if you want to see how I got auth to work. I had to replace `bcrypt` with `bcryptjs` npm package as it was complaining it couldn't find `bcrypt_lib.node` whenever I tried to import it. This seems to be caused by bcrypt not being able to download a precompiled bcrypt lib and the fallback failing to build from source.
